### DISCUSSION TITLE
Google Cloud run and Google Cloud Sql limitations

### DISCUSSION THREAD

**Discussion by ftognetto** [vercel/community#5636] [Help]
### Summary

Hello everyone, I have a sveltekit app deployed on Google Cloud Run that connects to a MySQL db on Google Cloud Sql.
I use Prisma as orm and since I have an architecture with multiple databases I have to instantiate several PrismaClients which consume many connections.
I'm having trouble on scaling up the application because even managing accurately those PrismaClients Google Cloud Run has a limit of 100 connections per instance https://cloud.google.com/sql/docs/quotas

So here's the question: if I deploy the app on Vercel do I have to expect the same limitation?

thanks,
Fabrizio

### Steps to Reproduce

https://cloud.google.com/sql/docs/quotas

**Comment by BartBoxi** [comment]
Hey @ftognetto,

if you using cloud run you will always get this quota limit of 100 connections per cloud run service. 

However, as far as I know, Vercel got similar quota limit - https://vercel.com/docs/storage/vercel-postgres/faq correct me if I'm wrong but I think limit would be the same. 

What you can try to do - you can try to connect with GCP Support Team and ask if you can increase quota for your project to ensure you can have more connections accepted for your cloud run service.

Hope it helps!
